src/main/scala/chisel3/util/experimental/BoringUtils.scala

```diff
@@ -160,6 +161,7 @@ object BoringUtils {
     * @param forceExists if true, require that the provided `name` parameter already exists in the global namespace
     * @throws BoringUtilsException if name is expected to exist and it doesn't
     */
+  @deprecated("Please use the new Boring API instead (BoringUtils.bore(source)). This will be removed in Chisel 7.0", "Chisel 7.0")
```

Comment:
```suggestion
  @deprecated("Please use the new Boring API instead (BoringUtils.bore(source)). This will be removed in Chisel 7.0", "Chisel 6.0")
```